### ISSUE TITLE
CLI: Provide user-facing error messages instead of Rust error stack

### ISSUE THREAD

**Issue by nicholastmosher** [infinyon/fluvio#356]
Right now our errors are nicely collected and propagated, but then they are just printed to the user. The CLI should have a set of specially crafted user-facing error messages to let them know what went wrong and how to fix it.

Below are some instances of error conditions which should have prettier errors:

- Error from `fluvio topic list` with "local" profile if local cluster is not installed
<img width="706" alt="image" src="https://user-images.githubusercontent.com/4210949/95856806-a3081180-0d28-11eb-91d5-08c900262f11.png">

**Comment by sehz** [comment]
Some additional clarification:

We should not display stack trace for any error or condition that we understood.  Examples are: 
- No Kubernetes connection
- No helm
- No Fluvio sys chart
- etc.

Also, we should not label them "Error" since this is type of condition we understood

If we encountered any error that we couldn't process or understood, we should display stack trace and output to stderr.

**Comment by sehz** [comment]
@ajhunyady provide feedback on how we should format user facing message

**Comment by nicholastmosher** [comment]
For reference, this is what is in the works which is intended to be the improvement to this: https://github.com/infinyon/fluvio/pull/489

**Comment by sehz** [comment]
So in case of pre-check failure, here is suggested format:
```
❯ fluvio cluster install
Performing pre-flight checks
✔️  ok: Kubernetes config is loadable
✔️  ok: Supported helm version is installed
 X    failed: No Kuberneted installation detected

Suggestion: Make sure you have run 'minikube start --driver=hyperkit'

```

**Comment by nicholastmosher** [comment]
I like that output for pre-checks. I do think that pre-check "check failed" rendering is actually a slightly different task than the one originally described by this issue. E.g. the user-printing strategy for the `fluvio topic list` error would be different than the user-printing strategy for `fluvio cluster install` pre-check failures.

**Comment by nicholastmosher** [comment]
Closed by #489
